confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -259,16 +259,16 @@ def test_compatibility(self, subject, avro_schema, version='latest'):
         try:
             result, code = self._send_request(url, method='POST', body=body)
             if code == 404:
-                log.error(("Subject or version not found:" + str(code)))
+                log.error("Subject or version not found:" + str(code))
                 return False
             elif code == 422:
-                log.error(("Invalid subject or schema:" + str(code)))
+                log.error("Invalid subject or schema:" + str(code))
                 return False
-            elif code >= 200 and code <= 299:
+            elif 200 <= code <= 299:
                 return result.get('is_compatible')
             else:
-                log.error("Unable to check the compatibility")
-                False
+                log.error("Unable to check the compatibility:" + str(code))
```

Comment:
Should be a space after ":"

Reply:
I also thought so but intentionally did not add it so it would be consistent with the two prints above